lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -3853,6 +3853,7 @@ Tag('ltx:*', 'afterClose:late' => sub {
 # These will get filled in during postprocessing.
 # * comes from hyperref
 DefConstructor('\ref OptionalMatch:* Semiverbatim', "<ltx:ref ?#1(class='ltx_nolink')() labelref='#label' _force_font='true'/>",
+  sizer      => '()',                                     # Don't actually know how big this will be!
```

Comment:
How do the empty parens get interpreted here? I haven't seen them used for sizer before.

Reply:
Not really interpreted at all; If the sizer is a string, its accumulated size is the size of the Whatsit; if  it contains any  '#digit`, the corresponding arg is "inserted".  So, basically the `\ref` has the same size as "()".